src/components/Modals/Onboarding/OnboardingBasicSingleDropdown.vue

```diff
@@ -61,32 +65,54 @@ export default defineComponent({
     cannotBeRemoved: { type: Boolean, required: true },
     scrollBottomToElement: { type: Number, default: 0 },
   },
+  mounted() {
+    this.curQuery = this.availableChoices[this.choice];
+    this.prevQuery = this.curQuery;
+  },
   emits: {
     'on-select': (acronym: string) => typeof acronym === 'string',
     'on-remove': () => true,
   },
   data() {
     return {
       shown: false,
+      curQuery: '',
+      prevQuery: '',
       stopClose: false,
       boxBorder: '',
       arrowColor: '',
       placeholderColor: this.choice !== '' ? lightPlaceholderGray : '',
     };
   },
+  computed: {
+    foundChoices(): [string, string][] {
+      return Object.entries(this.availableChoices).filter(v => v[1].startsWith(this.curQuery));
```

Comment:
Can we modify this to allow lowercase?

Right now, it seems like it has to be the exact match:

https://user-images.githubusercontent.com/54298311/118371588-3df4f280-b57b-11eb-9d13-b106e765a89b.mov